### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by YuigaWada**
## README
- ライブラリのバージョン・ビルドを明記.
- ライセンスを追加.

## 
READMEを更新しました.

実際にこのinstructionsでrun.pyとtrain.pyが動くか試してもらえますか？
ビルドの都合上, pythonのバージョンを3.7.11に変更しています.

### CODE REVIEW

**Comment by OtsuKotsu** [comment]
**これ多分main branchにpushしちゃだめです。**

> Repoの操作に関して返信がきたので共有します。
> > Yes, you can update the README of the repository to include the information mentioned in the comments.
I am using the current version of the repository to install and test the solution on the private test set. If you need to make some changes for adding a few analyses on the paper related to the training data, I encourage you to create a new branch from the main one.
Also, I noticed that the csv file contains estimations for the container mass of each configuration, but the paper does not discuss how you solved the estimation of that physical property. Please correct me if I am wrong. Otherwise, could you please add one or two paragraphs to illustrate the idea?
> 
> - READMEは更新してよい
> - トレーニングデータの分析を行うためにコードに変更を加えるのであれば、新たなブランチを作ること
> - Task 4（container mass estimation）の手法にも触れること
> 要約するとこんなところでしょうか。 （編集済み） 

**main-v2 branchなどを新たに切っておいて、そちらにpushの方が良いかも**

**Review by OtsuKotsu — APPROVED** [review]
現在`test.py`を回しています。今の所エラーは出ていません。

多分終了するまで回すことはできないので、冒頭まで`train.py`を回したときの出力を共有します。エラーは特に出ていません。

`python train.py ./train --task4`
```
Starting Task3 and Task5...

[task3.inference] 0 / 683
[video-processing] best frame is 195
[video-processing] best frame is 195
id=000000: cannot measure

[task3.inference] 1 / 683
[video-processing] NOT FOUND BEST FRAME ... !
[video-processing] best frame is 30
id=000001: cannot measure

[task3.inference] 2 / 683
[video-processing] best frame is 0
^C
```

`python train.py ./train --task1and2`
```
Training Task1 and Task2 ...

{'train loss': 1.1370962204519381, 'train accuracy': 0.4644628099173554, 'val loss': 0.9516559376930579, 'val accuracy': 0.6666666666666666}
{'train loss': 0.9297842647911103, 'train accuracy': 0.4446280991735537, 'val loss': 0.9563899620985373, 'val accuracy': 0.3974358974358974}
^C
```
